Makefile

```diff
@@ -0,0 +1,15 @@
+SHELL=/bin/bash
+
+lint:
+	python setup.py flake8
```

Comment:
I'm not too familiar with setuptools, but do you need to add `flake8` to `setup_requires` in `setup.py` in order to make this call?

Reply:
No, that would not be correct as it's a test dependency, not a runtime dependency. I added the test dependencies to `test_requires` in setup.py instead, but in practice it doesn't matter since I don't use setup.py to install the test dependencies. I just install it in the CI config file.